api/schemas.py

```diff
@@ -0,0 +1,103 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class LocationBase(BaseModel):
+    latitude: float
+    longitude: float
+    marked: bool
+
+
+class Location(LocationBase):
+    id: int
+    client_id: int
+
+    class Config:
+        orm_mode = True
+
+
+class ClientBase(BaseModel):
+    name: str
+
+
+class Client(ClientBase):
+    id: int
+    locations: List[Location] = []
+
+    class Config:
+        orm_mode = True
+
+
+class OrderBase(BaseModel):
+    location_origin_id: int
```

Comment:
can you change this to `location_origin_id: Tuple[float, float]`

Reply:
Looks like this will be set to the id of a Location() which inherits LocationBase which contains Lat&Lon points, so I think we're good. @Zhenia-Magic correct me if I'm reading it wrong.

Reply:
it's an id, not the location coordinates, I think it's okay.